### PR TITLE
README update

### PR DESCRIPTION
**Pull request by Ashley-Stanbridge**
Updated the readme do include running the server

### CODE REVIEW

File: server.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+clone the repo by running git clone https://github.com/common-io/common-platform.git
+
+once the repo has finished cloning run
+
+#### setvirtualenvproject
+
+once the project is set run
+
+#### pip  install -r requirements.txt
```
Comment (darylantony) [inline server.md]: The requirements currently assume that Postgres is installed.

File: README.md [hunk 2]
```diff
@@ -11,12 +11,47 @@ The Common Platform serves up the following channels:
...
 
+
 Which compromise mostly of static templates, indispersed with api calls to populate events, jobs and articles.
 
 Whilst pulling data from:
 
 + Common API    api.common.io
 
+
+##Runing dev server
```
Comment (darylantony) [inline README.md]: Change to "Local Installation"

File: README.md [hunk 3]
```diff
@@ -11,12 +11,47 @@ The Common Platform serves up the following channels:
...
+
+open your terminal and type
+
+#### mkvirtualenv --python=python3 NAME
```
Comment (darylantony) [inline README.md]: Please observe proper markdown syntax and used code blocks for any commands

**Comment by Ashley-Stanbridge** [comment]
Think this is all sorted?

File: README.md [hunk 4]
```diff
@@ -11,12 +11,53 @@ The Common Platform serves up the following channels:
...
+* When the repo has finished cloning run
+
+    `setvirtualenvproject`
+
+
+* Once the project is set run
+
+  Open repo in your text editor and comment out line 6 of requirements.txt
+
+  `<!-- psycopg2==2.6.2 -->`
```
Comment (darylantony) [inline README.md]: @Ashley-Stanbridge dewwwww this is a HTML comment.

Use `#` in python.

File: README.md [hunk 5]
```diff
@@ -11,12 +11,53 @@ The Common Platform serves up the following channels:
...
+    (where NAME is what you called your environment.)
+
+    To start the environment from anywhere.
+
+* Now run
+
+    `python manage.py runserver`
+
+
+* Visit http://localhost:8000/
```
Comment (darylantony) [inline README.md]: add backticks around the domain.

File: server.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
+
+## Hello!
```
Comment (darylantony) [inline server.md]: remove this file if its not needed now.

File: README.md [hunk 7]
```diff
@@ -11,12 +11,53 @@ The Common Platform serves up the following channels:
...
 Whilst pulling data from:
 
 + Common API    api.common.io
 
+
+##Local installation
+
+* Open your terminal and type,
+
+    ` mkvirtualenv --python=python3 NAME `,
```
Comment (darylantony) [inline README.md]: you only need to indent to get a code block; not indent and backtick.  backticks are for inline references to code variables.

File: README.md [hunk 8]
```diff
@@ -11,12 +11,53 @@ The Common Platform serves up the following channels:
...
 Which compromise mostly of static templates, indispersed with api calls to populate events, jobs and articles.
 
 Whilst pulling data from:
 
 + Common API    api.common.io
 
+
+##Local installation
+
+* Open your terminal and type,
```
Comment (darylantony) [inline README.md]: take away the bulleted list; and copy the style below.

**Comment by darylantony** [comment]
Merged in.  Feel free to delete the branch.

**Pull request merged**